__test__/services/roomTypeSerice.test.ts

```
@@ -0,0 +1,67 @@
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should add a room type', async () => {
+    const payload: AddRoomTypeRequestBody = {
+      name: 'Deluxe',
+      description: 'A deluxe room'
+    }
+    const mockInsertResult = { insertedId: new ObjectId() }(
+      databaseService.roomTypes.insertOne as jest.Mock
+    ).mockResolvedValue(mockInsertResult)
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)